wordpress-mobile/WordPress-Android
# Issue/13135 migrate post settings location picker

Fixes #13135

This PR replaces the deprecated PlacesPicker widget with the Autocomplete widget.

[![Image from Gyazo](https://i.gyazo.com/90aac7a50bdfc437b6e6d07397f550f0.png)](https://gyazo.com/90aac7a50bdfc437b6e6d07397f550f0)

Prerequisites:
- Since we rely on Google APIs, make sure to build the app with a valid debug certificate. Internal ref on how to setup one: paqN3M-9M-p2
- Make sure you are using wasabi flavor since it works with a debug certificate.

To test:
- Create a new post and Navigate to Post Settings.
- Tap on Location at the bottom of the screen.
- Confirm that the Autocomplete widget appears.
- Type the name of the landmark or street address. Make a selection.
- Confirm that the address of the place you picked is correctly set and displayed in the Location row.
- Publish the post, and reopen it. Navigate to Post Settings again.
- Confirm that the address is correctly displayed in the Location row.

Perform the above steps after removing or changing the address.

PR submission checklist:
